pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -462,6 +462,22 @@ type PMMSpec struct {
 	RuntimeClassName         *string                     `json:"runtimeClassName,omitempty"`
 }
 
+func (spec *PMMSpec) IsEnabled(secret *corev1.Secret) bool {
+	if !spec.Enabled || !spec.HasSecret(secret) {
+		return false
+	}
+	return true
```

Comment:
wouldn't it be simpler if:
```suggestion
	return spec.Enabled && spec.HasSecret(secret)
```

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1261/commits/7909d4c86535790c87cdacc43260b1ccc236dcc9